src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -201,6 +202,14 @@ struct BerdyOptions
     std::vector<std::string> jointOnWhichTheInternalWrenchIsMeasured;
 
     /**
+     * Name of the link which will be considered as a base frame.
+     *
+     * \note If the string is empty the default base link will be used,
+     *       i.e. the root of the tree
```

Comment:
The "root of the tree" comment is a bit confusing here, as in `iDynTree::Model` there is no root.